wagtailio/static/js/main.js

```diff
@@ -160,4 +160,23 @@ $(function() {
     })
   }
 
+  var $alert_container = $("[data-alert]");
+  if ($alert_container.data("alert")) {
+    $.getJSON($alert_container.data("alert"), function(data) {
+      if (data.text) {
+        $alert_container.html(data.text);
+        var styles = {
+          'top': $('.header')[0].getBoundingClientRect().bottom + 'px'
```

Comment:
For me this makes the alert display partly above the next element. Not an issue with the homepage but might be problematic elsewhere.